### PR TITLE
fix(styling): small changes to fix integration styling issues

### PR DESCRIPTION
**Pull request by velrest**

### CODE REVIEW

File: addon/templates/project/form.hbs [hunk 1]
```diff
@@ -7,14 +7,14 @@
       <UkSpinner @ratio="3" />
     </div>
   {{else}}
-    <h2 class="uk-flex uk-flex-between">
```
Comment (velrest) [inline addon/templates/project/form.hbs:10]: This already worked with alignment in the dummy app. Does kt-be have special button / h2 layout styling?

Comment (velrest) [reply addon/templates/project/form.hbs:10]: I also did a quick test in sz and seems to be ok like this.

Comment (anehx) [reply addon/templates/project/form.hbs:10]: No, this should be fine in BE as well.

**Review by anehx — APPROVED** [review]

**Pull request merged**